### PR TITLE
Update plot.cpp

### PR DESCRIPTION
**Pull request by sturkmen72**
patch for making **setPlotBackgroundColor** functional

### CODE REVIEW

**Comment by StevenPuttemans** [comment]
Euhm it seems reasonable that you would want to choose the background color yourself, but for now, you only change the function calling itself, not the actual implementation. There must be a part missing right?

**Comment by sturkmen72** [comment]
**setPlotBackgroundColor** defined before but not functional
https://github.com/sturkmen72/opencv_contrib/blob/update_plot_cpp/modules/plot/src/plot.cpp#L152-L155

**Comment by StevenPuttemans** [comment]
BUT what you are doing is incorrect, you are creating some sort of own matrix constructor, which is not what you want right?

File: modules/plot/src/plot.cpp [hunk 1]
```diff
@@ -182,7 +182,7 @@ namespace cv
             void render(Mat &_plotResult)
             {
                 //create the plot result
-                plotResult = Mat::zeros(plotSizeHeight, plotSizeWidth, CV_8UC3);
+                plotResult = Mat(plotSizeHeight, plotSizeWidth, CV_8UC3, plotBackgroundColor);
```
Comment (StevenPuttemans) [inline modules/plot/src/plot.cpp]: There is no MAT constructor that takes a Scalar as an extra argument inside OpenCV...

Comment (StevenPuttemans) [inline modules/plot/src/plot.cpp]: And the Mat::zeros is there to be sure that there is no random initialized crap inside the matrix...

Comment (sturkmen72) [inline modules/plot/src/plot.cpp]: http://docs.opencv.org/master/d3/d63/classcv_1_1Mat.html#a3620c370690b5ca4d40c767be6fb4ceb

Comment (StevenPuttemans) [inline modules/plot/src/plot.cpp]: Aha did not know about that :D

**Comment by alalek** [comment]
:+1:

**Pull request merged**